### PR TITLE
feat: delete PieChart from cost analysis

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

### CODE REVIEW

File: apps/web/src/services/cost-explorer/lib/config.ts [hunk 1]
```diff
@@ -23,13 +23,6 @@ export const MORE_GROUP_BY = {
     ADDITIONAL_INFO: 'additional_info',
 };
 
-export const GRANULARITY_ITEM_MAP = {
-    [GRANULARITY.ACCUMULATED]: { name: GRANULARITY.ACCUMULATED, label: 'Accumulated' },
-    [GRANULARITY.DAILY]: { name: GRANULARITY.DAILY, label: 'Daily' },
-    [GRANULARITY.MONTHLY]: { name: GRANULARITY.MONTHLY, label: 'Monthly' },
-    [GRANULARITY.YEARLY]: { name: GRANULARITY.YEARLY, label: 'Yearly' },
-} as const;
```
Comment (sulmoJ) [inline apps/web/src/services/cost-explorer/lib/config.ts:31]: question: Is it a variable that has never been used before?

**Review by sulmoJ — APPROVED** [review]
LGTM! 
I left a question!

File: apps/web/src/services/cost-explorer/lib/config.ts (hunk 1 above)
Comment (yuda110) [reply apps/web/src/services/cost-explorer/lib/config.ts:31]: Yes. I'm not sure why I made this😂

**Review by skdud4659 — APPROVED** [review]
LGTM!

**Pull request merged**